applications/volume_rendering_xr/utils/xr_hello_holoscan/README.md

```diff
@@ -24,9 +24,31 @@ The mixed reality demonstration scene includes:
 
 Refer to the [`volume_rendering_xr` README](/applications/volume_rendering_xr/README.md#prerequisites) for details on hardware, firmware, and software prerequisites.
 
-To run the application, run the following command in the HoloHub folder on your host machine:
+This utility is part of the `volume_rendering_xr` application suite. 
+
+### Default Behavior
+```bash
+./dev_container build_and_run volume_rendering_xr
+```
+This will launch the main volume rendering application by default.
+
+### Running the Test Utility
+To run this test utility instead:
+
+1. Open `/applications/volume_rendering_xr/metadata.json`
+2. Modify the `run` command to use `xr_hello_holoscan`:
```

Comment:
Rather than modifying `metadata.json`, could we handle trailing arguments from the [run launch --extra_args](https://github.com/nvidia-holoscan/holohub/blob/main/dev_container#L927) flag in `run.sh` out of the box?

A new user should be able to run `xr_hello_holoscan` without modifying files, sample command might look like this:
```bash
./run launch volume_rendering_xr --extra_args "xr_hello_holoscan"
```